strr-web/pages/registration-details/[id]/index.vue

```diff
@@ -85,21 +95,40 @@
                 {{ application?.unitAddress.postalCode || '-' }}
               </p>
               <p>
-                {{ application?.unitAddress.country ? regionNamesInEnglish.of(application?.unitAddress.country) : '-' }}
+                {{ application?.unitAddress.country
+                  ? regionNamesInEnglish.of(application?.unitAddress.country)
+                  : '-' }}
               </p>
             </BcrosFormSectionReviewItem>
+            <BcrosFormSectionReviewItem
+              :title="tApplicationDetails('parcelIdentifier')"
+              :content="application?.unitDetails.parcelIdentifier || '-'"
+              data-test-id="parcel-identifier"
+              class="break-all"
+            />
             <BcrosFormSectionReviewItem
               :title="tApplicationDetails('businessLicense')"
               :content="application?.unitDetails.businessLicense || '-'"
+              data-test-id="business-license"
               class="break-all"
             />
+            <BcrosFormSectionReviewItem :title="tApplicationDetails('listingLink')">
+              <a
+                :href="applicationDetails?.listingDetails[0].url"
+                target="_blank"
+                rel="noopener noreferrer"
+                class="text-ellipsis overflow-hidden break-words"
+                data-test-id="platform-url-1"
+              >
+                {{ applicationDetails?.listingDetails[0].url }}
+              </a>
+            </BcrosFormSectionReviewItem>
             <BcrosFormSectionReviewItem
-              v-if="application?.unitDetails.businessLicenseExpiryDate"
+              v-if="applicationDetails?.unitDetails.businessLicenseExpiryDate"
```

Comment:
Where is `applicationDetails` coming from?

Reply:
sorry while i was fixing this section i made a typo in 3 location for this. It has been now resolved.

Reply:
it should just say application?

Reply:
From the looks of it, it should be `application`. You are not verifying the values in local env?